docs/content/v2025.1/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -218,7 +218,7 @@ You can manage LDAP roles in YugabyteDB Anywhere using a combination of the foll
 
 You can map LDAP groups to [YugabyteDB Anywhere roles](../anywhere-rbac/#users-and-roles). After signing in, the user's role is assigned based on whatever role their LDAP group has been mapped to in YugabyteDB Anywhere. If a user is in multiple mapped LDAP groups, YugabyteDB Anywhere performs a union of all the roles that are chosen as a result of group mapping and assigns the union to the user.
 
-{{<tags/feature/ea>}}You can map groups to [fine-grained](../anywhere-rbac/#fine-grained-rbac) YugabyteDB Anywhere roles. To enable the feature in YugabyteDB Anywhere, set the **Enable RBAC for Groups** Global Runtime Configuration option (config key `yb.security.group_mapping_rbac_support`) to true. Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/). Note that only a Super Admin user can modify Global configuration settings.
+{{<tags/feature/ea idea="1015">}}You can map groups to [fine-grained](../anywhere-rbac/#fine-grained-rbac) YugabyteDB Anywhere roles. To enable the feature in YugabyteDB Anywhere, set the **Enable RBAC for Groups** Global Runtime Configuration option (config key `yb.security.group_mapping_rbac_support`) to true. Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/). Note that only a Super Admin user can modify Global configuration settings.
```

Comment:
IDEA-1015 is already GA inn 2024.1: https://yugabyte.atlassian.net/jira/polaris/projects/IDEA/ideas/view/5942094?selectedIssue=IDEA-1015